vita3k/modules/SceJpeg/SceJpegUser.cpp

```diff
@@ -150,7 +197,7 @@ EXPORT(int, sceJpegMJpegCsc, uint8_t *rgba, const uint8_t *yuv,
     uint32_t width = size >> 16u;
     uint32_t height = size & (~0u >> 16u);
 
-    convert_yuv_to_rgb(yuv, rgba, width, height, false);
+    convert_yuv_to_rgb(yuv, rgba, width, height, convert_color_space_jpeg_to_decoder((SceJpegColorSpace)(SCE_JPEG_COLORSPACE_YUV | sampling)));
```

Comment:
```suggestion
    convert_yuv_to_rgb(yuv, rgba, width, height, convert_color_space_jpeg_to_decoder(static_cast<SceJpegColorSpace>(SCE_JPEG_COLORSPACE_YUV | sampling)));
```
Small nitpick, can you use c++ casts instead of c cast?